app-backend/common/src/main/scala/S3.scala

```diff
@@ -0,0 +1,37 @@
+package com.azavea.rf.common
+
+import scala.util.{Success, Failure, Try}
+
+import org.apache.commons.io.IOUtils
+
+import com.amazonaws.auth._
+import com.amazonaws.services.s3.{AmazonS3Client, AmazonS3URI}
+import com.amazonaws.services.s3.model.{S3Object, ObjectMetadata}
+
+import java.io._
+import java.net._
+
+package object S3 {
```

Comment:
@notthatbreezy had suggested it might be a good solution to encapsulate smaller common functionality such as rollbar integration